src/app/series/directives/series-podcast.component.html

```diff
@@ -65,14 +65,17 @@
       <div class="fancy-hint">A link to the homepage for your podcast.</div>
     </publish-fancy-field>
 
-    <publish-fancy-field label="New Feed Url" textinput [model]="podcast" name="newFeedUrl">
-      <div class="fancy-hint">Use this to manually change the URL where your podcast is located.</div>
-    </publish-fancy-field>
-
     <publish-fancy-field label="Podcast Audio" [select]="audioVersionOptions" [model]="distribution" name="versionTemplateUrl" required>
       <div class="fancy-hint">Select which audio version template should be used with your podcast.</div>
     </publish-fancy-field>
 
+    <publish-fancy-field label="New Feed Url" textinput [model]="podcast" name="newFeedUrl" [advanced]="newFeedUrlConfirm">
```

Comment:
Since we're apparently going to be adding this to many fields, i wonder if this interface can be simplified a bit.

The fancy-field already has the model and field-name, so all it really needs is the warning message.  Something like `[confirm]=confirmNewFeedUrlMsg`.  I'm assuming the message string is long enough that it's not worth cramming directly into the element attribute.

Or you might be able to something tricky with a child `<div class="fancy-confirm">Are you sure?</div>`.  But I'm not confident that's actually possible.